model_kubeconfig.go

```diff
@@ -1,49 +1,49 @@
 package arcaflow_lib_kubernetes
 
 type KubeConfig struct {
-    Kind           string              `json:"kind"`
-    APIVersion     string              `json:"apiVersion"`
-    Clusters       []KubeConfigCluster `json:"clusters"`
-    Contexts       []KubeConfigContext `json:"contexts"`
-    Users          []KubeConfigUser    `json:"users"`
-    CurrentContext *string             `json:"current-context"`
-    Preferences    any                 `json:"preferences"`
+	Kind           string              `json:"kind" yaml:"kind"`
```

Comment:
I was using this structure to unmarshal the kubeconfig yaml file since the Parse() stub was returning this type, so I added the yaml tags. is that wrong?

Reply:
took me a while, but I got your point, stay tuned.